pkg/kubelet/kubelet.go

```diff
@@ -1143,8 +1144,8 @@ type Kubelet struct {
 	// It should be set only when docker is using non json-file logging driver.
 	dockerLegacyService legacy.DockerLegacyService
 
-	// StatsProvider provides the node and the container stats.
-	*stats.StatsProvider
+	// Provider provides the node and the container stats.
```

Comment:
this must have cause linter errors as name in comment doesn't match the name of a field:

```suggestion
	// StatsProvider provides the node and the container stats.
```

Reply:
@AndersonQ please address this comment